packages/ra-ui-materialui/src/input/SelectInput.spec.tsx

```diff
@@ -130,6 +131,14 @@ describe('<SelectInput />', () => {
             expect(options.length).toEqual(6);
             expect(options[1].textContent).toEqual('Created');
         });
+
+        it('should accept strings as choices', () => {
```

Comment:
I only tested SelectInput because the logic is shared by all selection inputs via `useChoicesContext`.